src/views/Item/Item.jsx

```
@@ -202,17 +310,50 @@ const Item = () => {
+                                    {/* Add your loader/spinner here */}
+                                    <Spinner size="sm" />
+                                </div>
+                            ) : (
+                                <>
+                                    {isFilterApplied && hasSelectedFilters && (
+                                        <div className="mb-4 flex items-center space-x-4">
+                                            {/* 'All' button */}
+                                            <button
+                                                className="px-4 py-2 bg-gray-200 hover:bg-gray-300 rounded-md"
+                                                onClick={clearFilters}
+                                            >
```

Biome

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)